libkpod/config.go

```diff
@@ -0,0 +1,38 @@
+package libkpod
+
+// Config stores the configuration options for libkpod
+type Config struct {
+	Root                     string
+	RunRoot                  string
+	Storage                  string
+	StorageOptions           []string
+	LogDir                   string
+	Runtime                  string
+	RuntimeUntrustedWorkload string
+	DefaultWorkloadTrust     string
+	Conmon                   string
+	ConmonEnv                []string
+	CgroupManager            string
+	DefaultTransport         string
+	InsecureRegistries       []string
+	SignaturePolicyPath      string
+}
+
+// DefaultConfig returns a Config struct with all options set to the default
+func DefaultConfig() *Config {
```

Comment:
The default for a lot of thee fields should come out of the library. Containers/storage now has a /etc/containers/storage.conf which incidates the default place an admin will store configuration.  
Root directory below and RunRoot should definitely come from there.

I think we want to get similar defaults out of the containers/image at some point.  

We don't want users to have to configure buildah, skopeo, kpod, cri-o all differently.  We need to have defaults in the libraries so that they set it once and all of the commands follow the rules, unless the command itself wants to override.